app/controllers/triage_controller.rb

```diff
@@ -21,6 +21,12 @@ def index
   def show
   end
 
+  def create
+    @triage = Triage.new(campaign: @session.campaign)
+    @triage.update!(triage_params)
+    redirect_to session_triage_index_path(@session)
```

Comment:
Should we handle potential errors here at this stage, or add it in later?

Reply:
You're right. Have you got an example of how to do this to hand?

Reply:
Done.